src/Microsoft.AspNet.Razor/CodeGenerators/CSharpCodeWriter.cs

```diff
@@ -63,6 +63,11 @@ public new CSharpCodeWriter WriteLine()
             return (CSharpCodeWriter)base.WriteLine();
         }
 
+        public CSharpCodeWriter WriteEndLine()
```

Comment:
Propagates lousy `bool endLine` parameter naming. Should be `WriteEndStatement()` and should probably be used in this class as well.